src/frontend/src/lib/components/hero/about/AboutItem.svelte

```diff
@@ -12,7 +12,9 @@
 >
 	<div class="flex items-center gap-2">
 		{#if asMenuItem}
-			<IconInfo />
+			<slot name="icon">
+				<IconInfo />
```

Comment:
i would say to remove the `IconInfo` from here, leave the slot and add the `IconInfo` in the slot directly inside `AboutHow` too (similar to `AboutWhat`)

WDYT?

Reply:
I agree. May I suggest a renaming of the component? Because this altered version would be better described by: `IconItem`. WDYT?

Reply:
I meant: AboutItem => IconItem

Reply:
Maybe more `ButtonAbout` or `AboutButton` if you want to rename it, since there is both text icon, no?